ct.yaml

```diff
@@ -3,3 +3,6 @@ chart-dirs:
 validate-maintainers: false
 remote: origin
 target-branch: main
+chart-repos:
+  - sigstore=https://sigstore.github.io/helm-charts
+  
```

Comment:
nit: extra line here

Reply:
It is a good practice to end files with a whiteline. especially useful when `cat` in the terminal.